src/components/AnimateSharedLayout/SharedLayoutContext.ts

```diff
@@ -97,3 +97,10 @@ export function isSharedLayout(
 export const SharedLayoutContext = createContext<
     SyncLayoutBatcher | SharedLayoutSyncMethods
 >(createBatcher())
+
+/**
+ * @internal
+ */
+export const FramerTreeContext = createContext<
```

Comment:
After seeing these again next to each other, I think `FramerTreeLayoutContext` might be a better name. WDYT @mattgperry ?